### DISCUSSION TITLE
How can I make the documentation of a personal alias?

### DISCUSSION THREAD

**Discussion by otakutyrant** [nushell/nushell#10047] [Q&A] [answered]
When putting a comment line before a command, the line will be shown as usage. But this rule seems do not apply to alias.

![2023-08-18-213647_1923x1080_scrot](https://github.com/nushell/nushell/assets/64188229/400d50cc-73b6-42ac-bdfa-31c5e3aa7f86)

In the image, the usage of `ll` is empty.

Maybe the feature is not implemented yet?

**Comment by fdncred** [comment, marked as answer]
Have you tried on the latest main branch? There's been some work in this area.

I have this in my config.nu
![image](https://github.com/nushell/nushell/assets/343840/b35ed585-4cfa-4c2c-8637-61e3f5b8541b)
The other aliases in this screenshot need to be fixed. I think they're broken because of recent changes.
![image](https://github.com/nushell/nushell/assets/343840/90fcd5ca-7786-421d-9e63-7e238b133604)

**Reply by otakutyrant** [reply]
I tried to complie the master version and it failed to pass one hundred tests...

I decide to put the comment line before alias anyway.

**Reply by fdncred** [reply]
hmm, weird that you'd have problems compiling. i compile nushell every day. This is the command I use after I've cloned the repository.
```
cargo install --path . --features=extra,dataframe,mimalloc --locked
```

**Reply by panicbit** [reply]
I recently needed to use `--locked` for it to compile via `cargo install`

**Reply by fdncred** [reply]
You're right. I do that too and just didn't say it. Lemme fix that.

**Reply by otakutyrant** [reply]
@fdncred I was installing nushell-git via AUR and failed to pass tests even with `--nocheck`. I am not familiar with Rust and Cargo so I give up installing Nushell on the master branch. Thank you anyway.

**Reply by otakutyrant** [reply]
[0.84.0](https://github.com/nushell/nushell/releases/tag/0.84.0) has fixed the issue now. Thank you!
